modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -15,13 +17,21 @@ public class PulsarContainer extends GenericContainer<PulsarContainer> {
 
     public static final String METRICS_ENDPOINT = "/metrics";
 
+    /**
+     * See <a href="https://github.com/apache/pulsar/blob/master/pulsar-common/src/main/java/org/apache/pulsar/common/naming/SystemTopicNames.java">SystemTopicNames</a>.
+     */
+    public static final String TRANSACTION_TOPIC_ENDPOINT =
+        "/admin/v2/persistent/pulsar/system/transaction_coordinator_assign/partitions";
+
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("apachepulsar/pulsar");
 
     @Deprecated
-    private static final String DEFAULT_TAG = "2.2.0";
+    private static final String DEFAULT_TAG = "2.10.0";
 
     private boolean functionsWorkerEnabled = false;
 
+    private boolean transactions = false;
```

Comment:
```suggestion
    private boolean transactionsEnabled = false;
```

Just so it is symmetrical to `functionsWorkerEnabled`.